client/main.go

```diff
@@ -35,7 +35,7 @@ import (
 type MeasuredResponse struct {
 	timeBetweenFrames time.Duration
 	latency           time.Duration
-	bytes             int64
```

Comment:
generally, seemed unneccessary/undesirable to use signed types for counters that can't go negative.